spec/serializers/auth_serilalizer_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe AuthSerializer do
+  let(:user) { create(:user, auth_token: auth_token) }
+  let(:auth_token) { create(:auth_token) }
+  let(:debate) { auth_token.debate }
+  let(:serializer) { described_class.new(user, debate) }
+
+  describe '#to_h' do
+    subject { serializer.to_h }
+
+    it 'returns auth_token' do
+      expect(subject[:auth_token]).to eq(auth_token.value)
```

Comment:
after giving it a second thought i think we should reformat expectation as:

```ruby
it 'should include auth_token' do # or it 'includes auth_token'
  expect(subject).to include(auth_token: auth_token.value)
end
```

here is the [include matcher ref](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher)